server/src/framework.ts

```diff
@@ -0,0 +1,63 @@
+import {
```

Comment:
consider coming up with a more informative filename, framework is somewhat general

Reply:
👍